### PR TITLE
Version 0.0.2

### PR DESCRIPTION
**Pull request by tombeynon**
This PR covers all the main requirements for the Omnibus:

- Standardised node install
- Standardised config (ENV and shortcuts)
- Remote chain info ([Akash style](https://github.com/ovrclk/net) or [Sikkatech style](https://github.com/sikkatech/tm-networks))
- Easy Statesync 
- Snapshot recovery (backup still needs doing)
- Private key backup and restore to Filebase
- Examples for all supported chains, on Akash and locally
- Some general examples for running validators, statesync etc

I've been using this 'in production' for a while and generally it provides enough features to make running a validator on Akash possible with a bit of care. It's also ridiculously easy to add a new chain, assuming you can find seeds/peers!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
-ARG  PROJECT_BIN
-ARG  PROJECT_VERSION
-
-ENV COSMOS_OMNIBUS_PROJECT=$PROJECT
-ENV COSMOS_OMNIBUS_PROJECT_VERSION=$PROJECT_VERSION
-
-ENV NODE_HOME=/data
-
-COPY dist/$PROJECT_BIN /bin/node
+RUN git clone $REPOSITORY /data
```
Comment (boz) [inline Dockerfile:34]: non-blocker: Can we do a multi-stage build that produces an image that only has `/bin/$PROJECT_BIN` and `/run.sh` added to it?

File: Dockerfile [hunk 2]
```diff
@@ -1,29 +1,45 @@
...
+WORKDIR /data
+RUN git checkout $VERSION
 
-COPY run.sh /run.sh
+RUN make install
```
Comment (boz) [inline Dockerfile:38]: Does this work for all of the projects?

**Review by boz — APPROVED** [review]
Amazing, thank you @tombeynon!

File: Dockerfile (hunk 1 above)
Comment (tombeynon) [reply Dockerfile:34]: Completely agree, and I think worth including this before we merge. If only to make the image size smaller. I'll look at this later today

File: Dockerfile (hunk 2 above)
Comment (tombeynon) [reply Dockerfile:38]: It does and it's [as per Tendermint docs](https://docs.tendermint.com/master/introduction/install.html).. but that doesn't mean it's going to work for all chains. We could make the `make install` part dynamic with a `build_command` build arg now, or do it when we find a chain that needs it

File: Dockerfile (hunk 1 above)
Comment (tombeynon) [reply Dockerfile]: That worked well - Akash image down from 2.26GB to 519MB, and it runs parts of the build in parallel too. `run.sh` does need the aws binary and a few apt packages, but this is as small as I could get it.

**Pull request merged**